plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -16,7 +16,7 @@
         .put("description", "Jenkins Configuration as Code")
         .put("type", "object");
 
-    public static JSONObject generateSchema() {
+    public static JSONObject generateSchema() throws Exception {
```

Comment:
Breaks compatibility for API consumers. Not a big deal in this case, but you might want to mark the plugin as restricted

Reply:
Is the throws exception actually needed? I can't see a change that requires this?